bindings/libdnf5/common.i

```diff
@@ -134,6 +134,35 @@ del ClassName##__iter__
 #endif
 %enddef
 
+%define add_str(ClassName)
```

Comment:
I also didn't figure out how to make this working using the `%feature("python:slot" ...`, so let's keep it as it is. I consider this implementation simple enough.